pkg/network/parentutils/parentutils.go

```diff
@@ -6,9 +6,11 @@ import (
 	"strconv"
 
 	"github.com/rootless-containers/rootlesskit/v2/pkg/common"
+	"github.com/sirupsen/logrus"
 )
 
 func PrepareTap(childPID int, childNetNsPath string, tap string) error {
+	logrus.Debugf("PrepareTap")
```

Comment:
This debug message doesn't seem meaningful?

Reply:
Oh, yes. I'll remove this debug leftover.

Reply:
ping